ting.client.inc

```diff
@@ -605,11 +605,10 @@ function ting_get_request_factory() {
   if (!isset($request_factory)) {
     $url_variables = array(
       'search' => 'ting_search_url',
-      'scan' => 'ting_scan_url',
       'object' => 'ting_search_url',
       'collection' => 'ting_search_url',
-      'spell' => 'ting_spell_url',
       'recommendation' => 'ting_recommendation_url',
+      'suggestion' => 'ting_suggestion_url',
```

Comment:
This removes configuration and th form elements in the administration interfa<ce for the preceding services. That is fine but I do not see any new administration fields  being added. I guess the are now moved to `ting_search_autocomplete` and I am OK with that to keep this module manageble but it needs to be clear when reading the code.

Please rename the variable to indicate where it belongs using the module as a prefix e.g. `ting_search_autocomplete_suggestion_url` and add a comment stating why the variable is not placed in this module even though it is in fact used here.